schema.json

```diff
@@ -0,0 +1,496 @@
+                                                    }
+                                                }
+                                            }
+                                        },
+                                        "apt_get": {
+                                            "type": "array",
+                                            "items": {
+                                                "type": "object",
+                                                "additionalProperties": false,
+                                                "properties": {
+                                                    "name": {
+                                                        "type": "string"
+                                                    },
+                                                    "bin_name": {
+                                                        "type": "string"
+                                                    }
+                                                }
+                                            }
+                                        },
+                                        "check_only": {
+                                            "type": "array",
+                                            "items": {
+                                                "type": "object",
+                                                "additionalProperties": false,
+                                                "required": [
+                                                    "name"
+                                                ],
+                                                "properties": {
+                                                    "name": {
+                                                        "type": "string"
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    }
+                                },
+                                "is_requires_admin_user": {
+                                    "type": "boolean"
+                                },
+                                "is_always_run": {
+                                    "type": "boolean"
+                                },
+                                "is_skippable": {
+                                    "type": "boolean"
+                                },
+                                "run_if": {
+                                    "type": "string"
+                                },
+                                "timeout": {
+                                    "type": "integer"
+                                },
+                                "meta": {
+                                    "type": "object"
+                                },
+                                "inputs": {
```

Comment:
please validate input keys also

Reply:
the website's validation is applied: https://github.com/bitrise-io/bitrise-steplib/pull/2431/files#diff-76d5ffeac090c76ed5f4c581f02f3c53R368